bin/node/bench/src/txpool.rs

```diff
@@ -78,17 +78,13 @@ impl core::Benchmark for PoolBenchmark {
 			context.client.clone(),
 		);
 
-		let generated_transactions = self
+		let start = std::time::Instant::now();
+		let submissions = self
 			.database
 			.block_content(
 				BlockType::RandomTransfersKeepAlive.to_content(Some(100)),
 				&context.client,
 			)
-			.into_iter()
-			.collect::<Vec<_>>();
-
-		let start = std::time::Instant::now();
```

Comment:
This is changing the scope of the benchmark. I don't think we want to do that.

Reply:
Sorry, I missed that it is a benchmark. I reverted my change